content/develop/concepts/configuration/theming-fonts.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Customize fonts
+slug: /develop/concepts/configuration/theming-customize-fonts
+---
+
+# Customize fonts in your Streamlit app
+
+Streamlit lets you change and customize the fonts in your app. You can load font files from a public URL or host them with your app using [static file serving](/develop/concepts/configuration/serving-static-files).
+
+## Default Streamlit fonts
+
+Streamlit comes with [Source Sans](https://fonts.adobe.com/fonts/source-sans), [Source Serif](https://fonts.adobe.com/fonts/source-serif), and [Source Code](https://fonts.adobe.com/fonts/source-code-pro) fonts. These font files are included with the Streamlit library to prevent clients from downloading them from a third party. By default, Streamlit uses Source Sans for all text except inline code and code blocks, which use Source Code instead.
+
+To use these default faults, you can set each of the following configuration options to `"sans-serif"` (Source Sans), `"serif"` (Source Serif), or `"monospace"` (Source Code) in `config.toml`:
+
+```toml
+[theme]
+font = "sans-serif"
+headingFront = "sans-serif"
+codeFont = "monospace"
+[theme.sidebar]
+font = "sans-serif"
+headingFront = "sans-serif"
+codeFont = "monospace"
+```
+
+- `theme.font` sets the default font for all text in the app (except inline code and code blocks). This is `"sans-serif"` (Source Sans) by default.
+- `theme.headingFont` sets the default font for all headings in the app. If this is not set, Streamlit uses `theme.font` instead.
+- `theme.codeFont` sets the default font for all inline code and code blocks. This is `"monospace"` (Source Code) by default.
+- `theme.sidebar.font` sets the default font for all text in the sidebar. If this is not set, Streamlit uses `theme.font` instead.
+- `theme.sidebar.headingFront` sets the default font for all headings in the sidebar. If this is not set, Streamlit uses (in order of precedence) `theme.headingFont`, `theme.sidebar.font`, or `theme.font` instead.
+- `theme.sidebar.codeFont` sets the default font for all inline code and code blocks in the sidebar. If this is not set, Streamlit uses `theme.codeFont` instead.
```

Comment:
The sidebar explanations seem pretty redundant here, maybe we can leave these out or just say "You can also set all of these options for the sidebar separately".